src/numerics/include/metaphysicl/dualnumber.h

```diff
@@ -397,6 +397,12 @@ DualNumber_std_unary(floor, 0,)
 template <typename T, typename D> \
 inline DualNumber<T, typename D::template rebind<T>::other> \
 funcname(const DualNumber<std::complex<T>, D> & in) \
+{ \
+  return {std::funcname(in.value()), std::numeric_limits<double>::quiet_NaN()}; \
+} \
+template <typename T> \
+inline DualNumber<T> \
+funcname(const DualNumber<std::complex<T>> & in)    \
```

Comment:
Why is this not covered by the previous case?

Reply:
The signature of the other function is:
```c++
template <typename T, typename D>
inline DualNumber<T, typename D::template rebind<T>::other> 
funcname(const DualNumber<std::complex<T>, D> & in) 
```
This should substitution fail when `D` is a builtin type like `double` etc.